src/services/itemLogin/itemLogin.controller.ts

```diff
@@ -149,6 +149,7 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
           await itemLoginService.delete(tx, itemId);
           reply.status(StatusCodes.NO_CONTENT);
         } catch (e: unknown) {
+          fastify.log.error(e);
```

Comment:
You should be able to use the log object present on the handler input arguments no ?